lib/CXGN/Contact.pm

```diff
@@ -76,48 +77,58 @@ sub send_email {
     print STDERR "$subject\n\n$body";
     if ( $vhost_conf->get_conf('production_server') ) {
         if ( $vhost_conf->get_conf('disable_emails') ) {
-            print STDERR
-"CXGN::Contact: Configured as production server, but not configured to send emails; no email sent from $mailfrom to $mailto.\n";
+            print STDERR "CXGN::Contact: Configured as production server, but not configured to send emails; no email sent from $mailfrom to $mailto.\n";
         }
         else {
-            my %mail = (
-                To      => $mailto,
-                From    => $mailfrom,
-                Subject => $subject,
-                Body    => $body,
-            );
-            $mail{'Reply-To'} = $replyto;
-
-            # Use External STMP server, if smtp config values are found
+
             my $smtp_server = $vhost_conf->get_conf('smtp_server');
             my $smtp_port = $vhost_conf->get_conf('smtp_port');
             my $smtp_user = $vhost_conf->get_conf('smtp_user');
             my $smtp_pass = $vhost_conf->get_conf('smtp_pass');
+
+            # If SMTP config values are found use external SMTP server
             if ( $smtp_server and $smtp_port and $smtp_user and $smtp_pass ) {
-                my $smtp_from = $vhost_conf->get_conf('smtp_from');
-                if ( $smtp_from ) {
-                    $mail{'From'} = $smtp_from;
-                }
-                $mail{'server'} = $smtp_server . ":" . $smtp_port;
-                $mail{'auth'} = {
-                    user => $smtp_user,
-                    password => $smtp_pass,
-                    method => 'DIGEST-MD5 CRAM-MD5 PLAIN LOGIN'
-                };
-            }
 
-            if ( sendmail(%mail) ) {
-                print STDERR
-"CXGN::Contact: Email notification sent from $mailfrom to $mailto.\n";
-            }
-            else {
-                print STDERR "CXGN::Contact: UNABLE TO SEND EMAIL NOTIFICATION\n";
+              my ($mail,$error) = Email::Send::SMTP::Gmail->new(
```

Comment:
@bellerbrock can you please add this into the Build.Pl and also add it as a dependency for the breedbase_docker?

Reply:
GC, no problem